src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -505,40 +506,28 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
       return undefined;
     }
 
-    if (state === "off") {
-      return supportsFeature(stateObj, SUPPORT_TURN_ON)
-        ? [
-            {
-              icon: "hass:power",
-              action: "turn_on",
-            },
-          ]
-        : undefined;
-    }
+    const buttons: ControlButton[] = [];
 
-    if (state === "on") {
-      return supportsFeature(stateObj, SUPPORT_TURN_OFF)
-        ? [
-            {
-              icon: "hass:power",
-              action: "turn_off",
-            },
-          ]
-        : undefined;
+    if (state === "off" && supportsFeature(stateObj, SUPPORT_TURN_ON)) {
+      buttons.push({
+        icon: "hass:power",
+        action: "turn_on",
+      });
     }
 
-    if (state === "idle") {
-      return supportsFeature(stateObj, SUPPORTS_PLAY)
-        ? [
-            {
-              icon: "hass:play",
-              action: "media_play",
-            },
-          ]
-        : undefined;
+    if (state === "on" && supportsFeature(stateObj, SUPPORT_TURN_OFF)) {
+      buttons.push({
+        icon: "hass:power",
+        action: "turn_off",
+      });
     }
 
-    const buttons: ControlButton[] = [];
+    if (state === "idle" && supportsFeature(stateObj, SUPPORTS_PLAY)) {
```

Comment:
We will now add next and previous buttons in idle mode, we should not.

Reply:
![image](https://user-images.githubusercontent.com/5662298/77235171-e7c53300-6bb3-11ea-83f5-8fb1a1cdae9f.png)